### ISSUE TITLE
Speedometer and other gauge charts display

### ISSUE THREAD

**Issue by eduard93** [intersystems-community/DeepSeeWeb#166]
Issues:
- [x] threshold min/max values are not respected
- [x] only one meter is displayed
- [x] meter value is not displayed
- [x] label is ignored
- [x]  if range is 0-1 (with value in 0-1) it's not respected.

![image](https://user-images.githubusercontent.com/5127457/107683654-e06ab480-6cb2-11eb-9103-51895324cc2d.png)

![image](https://user-images.githubusercontent.com/5127457/107683704-eeb8d080-6cb2-11eb-9252-36fa4d6d7664.png)

To test:
1. Import and compile: [oee.xml](https://gist.githubusercontent.com/eduard93/6a05ca9f9fc6d1cfcaf4f3ff0c6ec35f/raw/6474a3d12fd12fac825f4fd11b493571f09de8b3/oee.xml)
2. Run:
```
do ##class(isc.oee.model.Equipment).Populate()
do ##class(%DeepSee.Utils).%BuildCube("OEE")
```

**Comment by eduard93** [comment]
Also if range is 0-1 it's not respected.

**Comment by eduard93** [comment]
Any news on that?

**Comment by gnibeda** [comment]
All done, except range min/max calculation.
Is somewhere algorithm for range min/max calculation if not set?

**Comment by gnibeda** [comment]
fixed in 3.1.4
